app/views/static_pages/home/_how_it_works.html.erb

```diff
@@ -2,27 +2,33 @@
   <div class="how-it-works-title">
     <div class="container">
       <h2>How it works</h2>
-      <p>This is the website we wish we had when we were learning on our own. Our curriculum includes the best resources we could find online and here it is.</p>
+      <p>This is the website we wish we had when we were learning on our own. We scour the internet looking for only the best resources to supplement your learning and present them in a logical order.</p>
     </div>
   </div> <!-- /.how-it-works-title -->
 
   <div class="how-it-works-tiles">
-    <div class="tile">
-      <%= image_tag 'img-learn.svg', class: 'how-it-works-image', alt: 'how-it-works illustration' %>
-      <h3>Learn</h3>
-      <p>Learn from a curriculum with the best curated online tutorials, blogs, and courses.</p>
+    <div class="container">
+      <div class="tile">
```

Comment:
This should have a `container` class

Check the markup of the other two tiles.